test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -350,6 +415,78 @@ test.skip
     await t.expect(identityPage.getIdentifierValue()).eql('Test Identifier');
   });
 
+test
+  .requestHooks(loopbackSuccessLogger, loopbackSuccessWithHttpsMock)('in loopback server approach, https loopback succeeds', async t => {
+    const deviceChallengePollPageObject = await setup(t);
+    await checkA11y(t);
+    await t.expect(deviceChallengePollPageObject.getBeaconSelector()).contains(BEACON_CLASS);
+    await t.expect(deviceChallengePollPageObject.getFormTitle()).eql('Verifying your identity');
+    await t.expect(deviceChallengePollPageObject.getFooterCancelPollingLink().exists).eql(true);
+    await t.expect(loopbackSuccessLogger.count(
+      record => record.response.statusCode === 500 &&
+        record.request.url.match(/randomorgid.authenticatorlocaldev.com:2000\/probe/)
+    )).eql(1);
+    await t.expect(loopbackSuccessLogger.count(
+      record => record.response.statusCode === 500 &&
+        record.request.url.match(/randomorgid.authenticatorlocaldev.com:6512\/probe/)
+    )).eql(1);
+    await t.expect(loopbackSuccessLogger.count(
+      record => record.response.statusCode === 500 &&
+        record.request.url.match(/randomorgid.authenticatorlocaldev.com:6513\/probe/)
+    )).eql(1);
```

Comment:
NIT: I think L425-436 can be simplified to 
await t.expect(loopbackSuccessLogger.count(
   record => record.response.statusCode === 500 &&
      record.request.url.match(/randomorgid.authenticatorlocaldev.com:(2000|6512|6513)\/probe/)
)).eql(1);